### PR TITLE
features/webui/repositories_spec: Stub request to opensuse.org

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: src/api/spec/bootstrap/features/webui/repositories_spec.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@
   describe 'Repositories' do
     before do
       login admin_user
+
+      fake_distribution_body = File.open(Rails.root.join('test/fixtures/backend/distributions.xml')).read
+
+      stub_request(:get, 'https://api.opensuse.org/public/distributions.xml').
+        to_return(status: 200, body: fake_distribution_body, headers: {})
```
Comment (bgeuken) [inline src/api/spec/bootstrap/features/webui/repositories_spec.rb]: This change would mean that we can drop, or shorten, a couple of cassettes, or not?

Comment (coolo) [reply src/api/spec/bootstrap/features/webui/repositories_spec.rb]: 2:

```
spec/cassettes/Bootstrap_Repositories/Repositories/add/delete_repository_from_distribution.yml:    uri: https://api.opensuse.org/public/distributions.xml
spec/cassettes/Bootstrap_Repositories/Repositories/add_repository_from_distribution.yml:    uri: https://api.opensuse.org/public/distributions.xml

Comment (bgeuken) [reply src/api/spec/bootstrap/features/webui/repositories_spec.rb]: Then why don't you update them in the same commit?

Comment (coolo) [reply src/api/spec/bootstrap/features/webui/repositories_spec.rb]: Well, because killing them all is my target - but if you wish, I can arrange it

Comment (coolo) [reply src/api/spec/bootstrap/features/webui/repositories_spec.rb]: removing them is enough actually

**Pull request merged**